frame/contracts/src/lib.rs~

```diff
@@ -0,0 +1,864 @@
+// This file is part of Substrate.
```

Comment:
Why do we need this file?

Reply:
We don't. It's removed. 
This file is probably related to file sync with a remote development env.